### ISSUE TITLE
[Crash Report] Failed first launch

### ISSUE THREAD

**Issue by BigmenPixel0** [bottlesdevs/Bottles#640]
**Describe the bug**
Sometimes the first time you launch Bottles, it crashes

**Installation**
 - Method: flatpak
 - Version: 2021.10.14-treviso-2

**To Reproduce**
1. Remove ~/.var/app/com.usebottles.bottles
2. Open Bottles
3. Perform welcome screen actions

**Expected behavior**
Waited for the welcome screen to close normally

**Desktop (please complete the following information):**
 - OS: ArchLinux
 - Version: none
 - Display server: Wayland
**Additional context**
```
2021-10-17 16:24:59 INFO: Runners path doens't exist, creating now.
2021-10-17 16:24:59 INFO: Bottles path doens't exist, creating now.
2021-10-17 16:24:59 INFO: Dxvk path doens't exist, creating now.
2021-10-17 16:24:59 INFO: Vkd3d path doens't exist, creating now.
2021-10-17 16:24:59 INFO: Nvapi path doens't exist, creating now.
2021-10-17 16:24:59 INFO: Temp path doens't exist, creating now.
2021-10-17 16:25:07 INFO: Bottles Started!
TypeError: Must be number, not NoneType
2021-10-17 16:25:09 WARNING: No dxvk found.
2021-10-17 16:25:09 WARNING: No vkd3d found.
2021-10-17 16:25:09 WARNING: No nvapi found.
2021-10-17 16:25:09 WARNING: No runners found.
2021-10-17 16:25:12 INFO: Installing component: [vaniglia-6.18].
**
Gtk:ERROR:../gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
Bail out! Gtk:ERROR:../gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
```

**Comment by mirkobrombin** [comment]
I have tried several times to reproduce but without results 😅

**Comment by BigmenPixel0** [comment]
> I have tried several times to reproduce but without results

maybe the liquorix kernel is to blame for something?

**Comment by mirkobrombin** [comment]
> Similar problem with bottles crashing during initial setup. Mine goes through the install process and than crashes while or after (idk which) installing vaniglia throwing the error zsh: segmentation fault bottles.

fixed some days ago, will be released with 2021.11.14. Feel free to open this issue again if the problem persist.

**Comment by sponji** [comment]
I had a similar problem on Arch, after clicking next+next+install it would just crash, but I repeated it for ~10 times and it suddenly went through and started downloading stuff.

```
(15:06:00) INFO Performing Bottles checks... 
(15:06:00) INFO Runners found:
- sys-wine-6.23


(15:06:01) INFO Bottles Started! 
(15:06:10) INFO Performing Bottles checks... 
(15:06:10) WARNING No dxvk found. 
(15:06:10) WARNING No vkd3d found. 
(15:06:10) WARNING No nvapi found. 
(15:06:10) INFO Runners found:
- sys-wine-6.23


(15:06:10) WARNING No runners found. 
(15:06:11) INFO Installing component: [dxvk-1.9.2]. 
(15:06:11) INFO Installing component: [dxvk-nvapi-v0.4]. 
(15:06:11) INFO Installing component: [vkd3d-v2.4]. 
**
Gtk:ERROR:../gtk/gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
Bail out! Gtk:ERROR:../gtk/gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
**
Gtk:ERROR:../gtk/gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
Bail out! Gtk:ERROR:../gtk/gtk/gtkcssinheritvalue.c:33:gtk_css_value_inherit_free: code should not be reached
[1]    49785 abort (core dumped)  bottles
```
